src/Microsoft.TemplateEngine.Core/Util/ProcessorState.cs

```diff
@@ -15,6 +17,7 @@ public class ProcessorState : IProcessorState
     {
         private static readonly ConcurrentDictionary<IReadOnlyList<IOperationProvider>, Dictionary<Encoding, Trie<OperationTerminal>>> TrieLookup = new ConcurrentDictionary<IReadOnlyList<IOperationProvider>, Dictionary<Encoding, Trie<OperationTerminal>>>();
         private static readonly ConcurrentDictionary<IReadOnlyList<IOperationProvider>, List<string>> OperationsToExplicitlySetOnByDefault = new ConcurrentDictionary<IReadOnlyList<IOperationProvider>, List<string>>();
+        private readonly Stream _temporaryTarget;
```

Comment:
I suggest to use the explicit type to make the intent more clear.
```suggestion
        private readonly MemoryStream _temporaryTarget;
```